.github/workflows/ci.yaml

```diff
@@ -91,7 +91,11 @@ jobs:
     # Run unit tests
     - name: Run unit tests
       run: |
-        dotnet test --logger trx LoRaEngine/test/LoRaWanNetworkServer.Test/*.csproj -r LoRaEngine/test/TestResults/  &&  dotnet test --logger trx LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaToolsTest/*.csproj -r LoRaEngine/test/TestResults/ && dotnet test --logger trx LoRaEngine/test/LoraKeysManagerFacade.Test/*.csproj -r LoRaEngine/test/TestResults/
+        dotnet test --configuration ${{ env.buildConfiguration }} --logger trx -r \
+          LoRaEngine/test/TestResults \
+          /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura \
+          --filter 'FullyQualifiedName~LoRaWanTest|FullyQualifiedName~LoRaWan.NetworkServer.Test|FullyQualifiedName~LoraKeysManagerFacade.Test' \
```

Comment:
[nit] we don't need to repeat `FullyQualifiedName` for each test class (didn't find documentation on this but tried locally) 
NB: I don't recommend removing fully the explicit filter [as is mentioned here](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test#:~:text=An%20expression%20without%20an%20%3Coperator%3E%20is%20automatically%20considered%20as%20a%20contains%20on%20FullyQualifiedName%20property%20(for%20example%2C%20dotnet%20test%20--filter%20xyz%20is%20same%20as%20dotnet%20test%20--filter%20FullyQualifiedName~xyz).) - only not to repeat it every time.

Reply:
> didn't find documentation on this but tried locally

If it's not documented then I'd rather not go with it if you don't mind. In fact, the filters didn't work reliably for me when _listing_, but that could be just bugs in the .NET Core SDK 3.1 tooling. I don't think we should be relying on the filtering on names in the longer run anyway. It would be much better to go with categories but I didn't want to expand the scope of this PR.